src/pages/admin/events/[eventId]/index.tsx

```diff
@@ -0,0 +1,288 @@
+import React, { useState } from "react";
+import { NextPage, NextPageContext } from "next";
+import Router from "next/router";
+import { Event, Volunteer, EventVolunteer, PaginatedVolunteers, APIError } from "utils/types";
+import UpsertEvent from "src/components/UpsertEvent";
+import { getEvent, getEventVolunteers } from "server/actions/Event";
+import {
+    Container,
+    Grid,
+    TextField,
+    InputAdornment,
+    TableContainer,
+    Paper,
+    Table,
+    TableHead,
+    TableRow,
+    TableCell,
+    TableBody,
+    Link,
+    makeStyles,
+    Theme,
+    createStyles,
+    Switch,
+    Button,
+} from "@material-ui/core";
+import { Search } from "@material-ui/icons";
+import { styles } from "@material-ui/pickers/views/Calendar/Calendar";
+import colors from "src/components/core/colors";
+import urls from "utils/urls";
+import { addVolunteer, markVolunteerNotPresent, markVolunteerPresent } from "server/actions/Volunteer";
+import VolAttendanceListItem from "src/components/VolAttendanceListItem";
+import CoreTypography from "src/components/core/typography";
+import VolQuickAddDialog from "src/components/VolQuickAddDialog";
+import InfiniteScroll from "react-infinite-scroll-component";
+import constants from "utils/constants";
+
+interface Props {
+    event: Event;
+    pageVols: PaginatedVolunteers;
+}
+
+function orderVolunteers(vols: Volunteer[]): Volunteer[] {
```

Comment:
Don't think this is used - do you think you can remove this?

Reply:
yea def can be removed